core/src/integration-test/java/com/scalar/db/api/DistributedStorageIntegrationTestBase.java

```diff
@@ -210,6 +213,8 @@ public void get_GetWithProjectionsGiven_ShouldRetrieveSpecifiedValues()
         .isEqualTo(Optional.of(new IntValue(COL_NAME3, pKey + cKey)));
     assertThat(actual.get().getValue(COL_NAME4).isPresent()).isTrue(); // since it's clustering key
     assertThat(actual.get().getValue(COL_NAME5).isPresent()).isFalse();
+    assertThat(actual.get().contains(COL_NAME6)).isTrue();
+    assertThat(actual.get().isNull(COL_NAME6)).isTrue();
```

Comment:
I updated some unit tests to be more robust by projecting as well the column `COL_NAME6` which value is null.